lib/eventsource.dart

```diff
@@ -91,6 +96,12 @@ class EventSource extends Stream<Event> {
         request.headers[k] = v;
       }); 
     }
+    if (_body == null) {
+      request.body = '';
+    } else {
+      request.body = json.encode(_body);
```

Comment:
Hmm, this is implying that all bodies will be JSON?

Doesn't it make more sense to have the `_body` field be a `String` and do the JSON encoding when calling `Eventsource.conncet`?

Reply:
Yes, you are right with your suggestions :)

I just implement what I needed fast and practically doing copy/paste with other code... I will correct this when I have time. 

But I will assume that all the bodies will be JSON (so only JSON bodies will be supported)

Reply:
Yeah so I would prefer to have this changed to allow any String as body and do the JSON conversion beforehand. Then I'd be glad to merge the fix. Thanks!

Reply:
Oh ok! I misunderstood you, now is corrected